src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -47,21 +47,22 @@ def config(tmp_path):
             "namelist": {
                 "update_values": {
                     "config": {
-                        "atm_core_files_input_grid": [str(afile), str(afile)],
+                        "fix_dir_target_grid": "/path/to/dir",
+                        "mosaic_file_target_grid": str(afile),
+                        "orog_dir_target_grid": "/path/to/dir",
+                        "orog_files_target_grid": str(afile),
+                        "vcoord_file_target_grid": str(afile),
+                        "varmap_file": str(afile),
+                        "data_dir_input_grid": str(afile),
                         "atm_files_input_grid": str(afile),
-                        "atm_tracer_files_input_grid": str(afile),
-                        "atm_weight_file": str(afile),
+                        "sfc_files_input_grid": str(afile),
+                        "grib2_file_input_grid": str(afile),
+                        "tracers_input": [str(afile), str(afile)],
+                        "tracers": [str(afile), str(afile)],
                         "convert_atm": True,
-                        "data_dir_input_grid": str(afile),
+                        "input_type": "gaussian_nemsio",
                         "external_model": "GFS",
-                        "fix_dir_target_grid": "/path/to/dir",
                         "geogrid_file_input_grid": str(afile),
-                        "grib2_file_input_grid": str(afile),
-                        "mosaic_file_input_grid": str(afile),
-                        "mosaic_file_target_grid": str(afile),
-                        "sfc_files_input_grid": str(afile),
-                        "varmap_file": str(afile),
-                        "vcoord_file_target_grid": str(afile),
```

Comment:
Please put this back in alphabetical order.

Reply:
Got it, thanks